data/Dockerfiles/acme/docker-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+ACME_BASE=/var/lib/acme
+mkdir -p ${ACME_BASE}/acme/private
+
+restart_containers(){
+	for container in $*; do
+		curl -X POST \
+			--unix-socket /var/run/docker.sock \
+			"http:/containers/${container}/restart"
+	done
+}
+
+while true; do
+
+	acme-client \
+		-v -b -N -n \
+		-f ${ACME_BASE}/acme/private/account.key \
+		-k ${ACME_BASE}/acme/private/privkey.pem \
+		-c ${ACME_BASE}/acme \
+		${MAILCOW_HOSTNAME} autoconfig.${MAILCOW_HOSTNAME#*.} autoconfig.${MAILCOW_HOSTNAME#*.} ${ADDITIONAL_SAN}
```

Comment:
It would be nice to be able to configure the full list. So you don't have to use `autoconfig`

Also the list ist wrong, it should be `autodiscover.${MAILCOW_HOSTNAME#*.} autoconfig.${MAILCOW_HOSTNAME#*.}` --> `discover` :wink: